models/curvefi/ethereum/curvefi_ethereum_trades.sql

```diff
@@ -79,7 +79,7 @@ WITH dexs AS
     FROM {{ source('ethereum', 'logs') }} l
     JOIN  {{ ref('curvefi_ethereum_view_pools') }} p
         ON l.contract_address = p.pool_address
-        AND p.version = 'Factory V2'
+        AND p.version IN ('Factory V2') --, 'Regular' ) --some Regular pools are new and use the below topic instead
```

Comment:
i'm not sure on this error -- for this change, can we try `AND (p.version = 'Factory V2' or p.version = 'Regular')` rather than `IN` to see if that's the issue? seems highly unlikely, but start with the simplest approach